crates/router/src/connector/globalpay/requests.rs

```diff
@@ -642,20 +638,6 @@ pub enum ChipCondition {
     PrevSuccess,
 }
 
-/// Card Verification Value Indicator sent by the Merchant indicating the CVV
-/// availability.
-#[derive(Debug, Default, Serialize, Deserialize)]
-#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
-pub enum CvvIndicator {
```

Comment:
Why are we removing this? Are we not getting this info in response?

Reply:
Default value of `CvvIndicator` is `PRESENT` , that means we must send cvv . For mandates we can't send Cvv for subsequent payments , as we don't store cvv .  I don't see any significance of `CvvIndicator` from hyperswitch side so i removed it .